openstf/feature_engineering/proloaf_historic_features.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+""" This module contains all proloaf related functions used for feature engineering.
```

Comment:
I think it is better remove proloaf from the name of this file as other models might need the same features in the future. However it is good to mention in the documentation that this feature is intended for models that generate their own lag features such as proloaf